django_rq/views.py

```diff
@@ -493,6 +494,17 @@ def actions(request, queue_index):
                 for job_id in job_ids:
                     requeue_job(job_id, connection=queue.connection)
                 messages.info(request, 'You have successfully requeued %d  jobs!' % len(job_ids))
+            elif request.POST['action'] == 'stop':
+                cancelled_jobs = 0
+                for job_id in job_ids:
+                    try:
+                        send_stop_job_command(queue.connection, job_id)
+                        job = Job.fetch(job_id, connection=queue.connection)
```

Comment:
You shouldn't need to manually cancel the job. Does RQ not already automatically put the job in `FailedJobRegistry`?

If that's not the case this should be a bug that needs to be fixed within RQ.

Reply:
@selwin Well, I was struggling a bit with the tests (to make it actually catch the job before it's marked as finished). 
But also, if you stop the job, you might not want to have the job in the failed column. I will be doing some tests so I'm happy to hold a bit. Also I will add the stop button that you suggested.

Reply:
@selwin I fixed as per your requests, stop job does not cancel it. it just goes to the Failed registry.
Also the button was added.
I think it's ready to be merged.